File: online-grocery-ordering-System-frontend/src/app/common/resturl.ts
const mainURL="http://localhost:8080/";

export const admin = "admin/";
export const user = "user/";

export const login = mainURL + 'authenticate';
export const registration = mainURL + 'register';

//profile
export const GET_USER_DETAILS = mainURL + user+'get-user-details';
export const USER_ADDRESS = mainURL + user+'address';
export const USER_ADDRESS_GET_ALL = mainURL + user+'address-get-all';
export const USER_ADDRESS_GET_BY_ID = mainURL + user+'address-get-by-id';

File: OnlineGroceryOrderingSystemAuthentication/src/main/kotlin/com/allion/ogoss/util/RestURL.kt
package com.allion.ogoss.util

open class RestURL {
    companion object {
        const val USER : String ="user/"
        val ADMIN : String ="admin/"

        const val REGISTER :String = "register"
        //profile
        const val GET_USER_DETAILS : String = USER+"get-user-details"
        const val USER_ADDRESS : String = USER+"address"
        const val USER_ADDRESS_GET_ALL : String = USER+"address-get-all"
        const val USER_ADDRESS_GET_BY_ID : String = USER+"address-get-by-id"



    }
}